core/loadpoint.go

```diff
@@ -55,7 +52,7 @@ const (
 	minActiveCurrent = 1.0 // minimum current at which a phase is treated as active
 	minActiveVoltage = 207 // minimum voltage at which a phase is treated as active
 
-	guardGracePeriod          = 60 * time.Second // allow out of sync during this timespan
+	enabledCommandTimeout     = 60 * time.Second // allow out of sync during this timespan
```

Comment:
```suggestion
	chargerEnableUpdated      = 60 * time.Second // allow out of sync during this timespan
```

Reply:
Ich würde gerne bei enabledCommandTimeout bleiben. Es gibt ja auch ein phaseSwitchCommandTimeout.

Reply:
Habe es jetzt umbenannt in enabledCommandDuration, analog zu phaseSwitchDuration